src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,17 @@
+package domain
+
+class Board(private val height: Int, private val width: Int) {
+    private val cells: MutableList<Cell> = MutableList(height * width) { index ->
+        Cell(Position(index % width, index / width))
+    }
```

Comment:
```suggestion
class Board(height: Int, width: Int) {
    private val cells: List<Cell> = List(height * width) { index ->
        Cell(Position(index % width, index / width))
    }
```

- height와 width는 생성자로만 써도 괜찮을 것 같아요! 
- cells에 대해서 현재 Board 안에서 변화되어지지 않은 것 같은데, 불변을 지키는 건 어떨까요?

Reply:
`MutableList` 가 가변이니 불변으로 하는 것도 괜찮은 것 같습니다.